robonomics/io/Cargo.toml

```diff
@@ -12,6 +12,8 @@ derive_more = "0.99"
 serialport = "3.3"
 async-std = "1.5"
 sds011 = "0.1"
+ipfs-api = "0.7.1"
```

Comment:
I recommend to use https://github.com/akru/rust-ipfs-api because another IPFS clients require OpenSSL library that looks a bit heavy.